generator/integration_tests/golden/golden_logging_decorator_test.cc

```diff
@@ -31,136 +31,135 @@ using ::testing::Return;
 
 class MockGoldenStub
     : public google::cloud::golden_internal::DatabaseAdminStub {
- public:
+public:
   ~MockGoldenStub() override = default;
   MOCK_METHOD(
       StatusOr<::google::test::admin::database::v1::ListDatabasesResponse>,
       ListDatabases,
       (grpc::ClientContext & context,
-       ::google::test::admin::database::v1::ListDatabasesRequest const&
-           request),
+       ::google::test::admin::database::v1::ListDatabasesRequest const
+           &request),
       (override));
 
   MOCK_METHOD(StatusOr<::google::longrunning::Operation>, CreateDatabase,
               (grpc::ClientContext & context,
-               ::google::test::admin::database::v1::CreateDatabaseRequest const&
-                   request),
+               ::google::test::admin::database::v1::CreateDatabaseRequest const
+                   &request),
               (override));
 
   MOCK_METHOD(
       StatusOr<::google::test::admin::database::v1::Database>, GetDatabase,
       (grpc::ClientContext & context,
-       ::google::test::admin::database::v1::GetDatabaseRequest const& request),
+       ::google::test::admin::database::v1::GetDatabaseRequest const &request),
       (override));
 
   MOCK_METHOD(
       StatusOr<::google::longrunning::Operation>, UpdateDatabaseDdl,
       (grpc::ClientContext & context,
-       ::google::test::admin::database::v1::UpdateDatabaseDdlRequest const&
-           request),
+       ::google::test::admin::database::v1::UpdateDatabaseDdlRequest const
```

Comment:
Some formatting looks strange w/ the placement of the ampersand. Also on the previous line where there are spaces on both sides of the ampersand.